.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
```

Comment:
if you add `sudo: false`, Travis will use their container-based testing which is reportedly faster: http://blog.travis-ci.com/2014-12-17-faster-builds-with-container-based-infrastructure/